confidential-assets/src/confidentialAsset.ts

```diff
@@ -146,8 +152,10 @@ export class ConfidentialAsset {
 
   async withdraw(
     args: CreateConfidentialWithdrawOpArgs & {
-      to?: AccountAddressInput;
       tokenAddress: string;
+      to?: AccountAddressInput;
```

Comment:
we should either make it a required field, or mention that if it is not set, it refers to the sender